tests/test_security_files.py

```diff
@@ -0,0 +1,7 @@
+def test_insecure_post_fails(flask_test_client):
+  response = flask_test_client.post(
+    '/',
+    data = dict(email="test@company.com", password="test"),
+    follow_redirects=True
+    )
+  assert response.status_code == 403
```

Comment:
Do we need to test both HSTS security (enforced TCP with the introduction of strict_transport_security) and CSRF preventing cross-site request forgery (with the introduction of SeaSurf) here?

Reply:
Could HSTS be tested by making a http (not https) request checking it fails? We are currently (very implicitly and manually) testing this when we attempt to use flask in production mode - since our local dev server is not https- because it fails all requests.

Reply:
yes i think the http test expecting failure would cover HSTS - always good to have the explicit test